tests/platform_helper/fixtures/make_addons/expected/web/addons/my-rds-db.yml

```diff
@@ -0,0 +1,347 @@
+# Generated by copilot-helper v0.1-TEST / 2023-08-22 16:00:00
+Parameters:
+  # Copilot required Parameters...
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+
+Mappings:
+  myRdsDbEnvironmentConfigMap:
+    development:
+      DeletionPolicy: Delete
+      SnapshotIdentifier: test-snapshot-id
+    production:
+      DeletionPolicy: Delete
+      SnapshotIdentifier: ""
+
+Conditions:
+  myRdsDbUseSnapshot: !Not [!Equals [!FindInMap [myRdsDbEnvironmentConfigMap, !Ref Env, SnapshotIdentifier], ""]]
+
+Resources:
+  myRdsDbLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub "${App}-${Env}-${Name}-myRdsDb-rds-create-user"
+      Handler: index.handler
+      Runtime: python3.11
+      Layers:
+        - arn:aws:lambda:eu-west-2:763451185160:layer:python-postgres:1
+      MemorySize: 128
+      Timeout: 10
+      Role: !GetAtt myRdsDbLambdaFunctionExecutionRole.Arn
+      VpcConfig:
+        SecurityGroupIds:
+          - !ImportValue
+            'Fn::Sub': "${App}-${Env}-myRdsDb-DBClusterSecurityGroupName"
+        SubnetIds: 
+          !Split 
+            - ","
+            - !ImportValue
+              'Fn::Sub': "${App}-${Env}-myRdsDb-PrivateSubnetsName" 
+      Code:
+        ZipFile: |
```

Comment:
Feels like there must be a better way of doing this rather than copy-pasting the lambda function from the source file every time there's a change. Maybe either making this bit a regex wildcard, or replacing a placeholder with the actual file content or something.

Reply:
Are you talking about the Terraform version, which will supersede this shortly?

Reply:
@antroy-madetech I don't mind the copy-pasting leg work because I want a unit test to fail when someone makes a change to the lambda code and to think what the change means while they're copying it over.

Reply:
Going forward we will be using https://github.com/uktrade/terraform-platform-modules/blob/main/postgres/manage_users.py, which I am about to create a ticket to add unit tests for if we don't have them already.